FhirToDataLake/build.yml

```diff
@@ -38,7 +38,21 @@ steps:
     docker pull healthplatformregistry.azurecr.io/fhir-analytics-data-source:v0.0.1
     docker run --rm -d -p 5000:5000 --name mock-data-source healthplatformregistry.azurecr.io/fhir-analytics-data-source:v0.0.1
   displayName: Start mock-data-source
-  
+
+- task: AzureKeyVault@2
+  inputs:
+    azureSubscription: 'ResoluteOpenSource'
+    KeyVaultName: 'fhiranalyticspipeline'
```

Comment:
What's this keyVault be used for? Can we use secret variables in pipeline?

Reply:
It's used to store secret variables . Yes, secrets can also be saved in pipeline variable. I think we can try it in later pull request.